client/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { h, render } from "preact";
+
+import Reporter from "./components/Reporter";
+
+import "./style/index.css";
+
+window.addEventListener("load", () => {
```

Comment:
we are going to bundle everything and inline the script at the end of the body. There are no external scripts or stylesheets. We can control the order of loading.